modules/server/plugins/home.php

```diff
@@ -1,39 +1,35 @@
 <?php
-
-$smarty->assign('caption', t('Neue Server'));
+$smarty->assign('caption', t('Server für die ') . ' ' . $_SESSION['party_info']['name']);
```

Comment:
```suggestion
$smarty->assign('caption', t('Server für ') . ' ' . $_SESSION['party_info']['name']);
```
Ich bin halb überzeugt, dass das Weglassen von "die" besser klingt, halb aber auch nicht.
Freigestellt jede Variante zu übernehmen, weil egal ;)